roles/preflight-checks/tasks/ceph.yml

```diff
@@ -26,3 +21,17 @@
   delegate_to: "{{ item }}"
   with_items:
     - "{{ groups['controller'] }}"
+
+- name: detect scale-out
```

Comment:
no one should skip precheck except the developers those who know the details they are skipping.
But if really the operator skipped precheck by mistake, the ceph_scale_out is default(false)